projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -737,13 +751,21 @@ function maybe_gate_existing_comments( $comment ) {
 /**
  * Placeholder text for non-subscribers
  *
+ * @param string $newsletter_access_level The newsletter access level.
  * @return string
  */
-function get_locked_content_placeholder_text() {
+function get_locked_content_placeholder_text( $newsletter_access_level ) {
+	$access_level = $newsletter_access_level === Token_Subscription_Service::POST_ACCESS_LEVEL_PAID_SUBSCRIBERS ? 'paid subscribers' : 'subscribers';
```

Comment:
Should we make this 2 strings translatable?

Reply:
Good catch! Fixed in [da399bf](https://github.com/Automattic/jetpack/pull/29860/commits/da399bfabdaf0f9c93db5092be80af83bf6a7e84)